src/pages/List/List.js

```diff
@@ -1,6 +1,80 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
+import { HotelList } from './HotelList';
 
 export const List = () => {
-  return <div></div>;
+  const [hotels, sethotels] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/listMockdata.json')
```

Comment:
```suggestion
    fetch('/data/listMockdata.json')
```
public 참조할때는 서버주소 일일이 적어줄 필요 없습니다.

Reply:
수정하였습니다.